src/utilities/identities/identities.ts

```diff
@@ -128,10 +130,15 @@ export function getKeypairBySeed(seed: Uint8Array): KiltKeyringPair {
   return makeKeyring().addFromSeed(seed) as KiltKeyringPair;
 }
 
+type GetStoreTxSignCallback = (
+  signData: Omit<SignRequestData, 'did'>,
+) => Promise<Omit<SignResponseData, 'keyUri'>>;
+
```

Comment:
Why doesn't the SDK export this type? The `SignCallback` type doesn't work with Did.Chain.GetStoreTx

Reply:
you can use the `Parameters` helper to get the type definition from the function.
But you are already the second person asking about this, so I think I rather export it.

Reply:
We had a look together with Luke and it seems this type is not needed at all

Reply:
Upon further inspection, we do actually need to define the type in the `IdentityDidCrypto` interface